starship

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+import pdb
+
+from strawberryfields.engine import StarshipEngine
+from strawberryfields.io import load
+from strawberryfields import configuration
+
+PROMPTS = {
+    "hostname": "Please enter the hostname of the server to connect to: [{}] ",
+    "authentication_token": "Please enter the authentication token to use when connecting: [{}] ",
+    "save": "Would you like to save these settings to a local cofiguration file in the current "
+    "directory? [Y/n] ",
+}
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="run Strawberry Fields code on StarshipEngine")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument("--input", "-i", help="the XBB file to run")
```

Comment:
Is this required? What happens if you run the script without this argument, you just get the help messages?

Reply:
Yes the output is:

```
usage: starship [-h] [--input INPUT] [--output OUTPUT] [--debug]
                [--reconfigure]
starship: error: one of the arguments --input/-i --reconfigure is required
```

in that case.